src/lib/signature_lib/schnorr.ml

```diff
@@ -494,12 +450,16 @@ module Message = struct
 
   let network_id_testnet = Char.of_int_exn 0
 
+  let network_id_other = Char.of_int_exn 2
```

Comment:
We want to achieve domain separation between each of these new domains too, but I don't think the mechanism is decided yet. Hard-coding 2 here is cryptographically dangerous.
Can you delete this, and put a `failwith "TODO"` where it's used below, to make sure this doesn't leak out into production?